src/board_controller/build.cmake

```diff
@@ -229,12 +251,8 @@ if (ANDROID)
 endif (ANDROID)
 
 if (UNIX AND NOT ANDROID)
-    target_link_libraries (${BOARD_CONTROLLER_NAME} PRIVATE pthread dl)
+    target_link_libraries (${BOARD_CONTROLLER_NAME} PRIVATE pthread dl m c)
 endif (UNIX AND NOT ANDROID)
-if (ANDROID)
-    find_library (log-lib log)
```

Comment:
why is it removed?

Reply:
Restored find_library for 'log-lib' to ensure proper Android linking.